src/main/groovy/com/rundeck/plugins/ansible/plugin/AnsibleResourceModelSource.java

```diff
@@ -142,6 +144,24 @@ private static String resolveProperty(
         }
   }
 
+  private static Integer resolveIntProperty(
```

Comment:
would it be possible to have the method "resolveProperty" to receive the class to cast the value to instead of having a new method? 🤔

Reply:
Luis requested to do that method like that.